man/runc-update.8.md

```diff
@@ -5,42 +5,41 @@
    runc update [command options] <container-id>
 
 # DESCRIPTION
-   The data can be read from a file or the standard input, the
-accepted format is as follow (unchanged values can be omitted):
+   The update command can be used to update resource constraints of a container.
+Values can be set by arguments, a file or the standard input.
 
+# OPTIONS
+   --resources, -r              path to the file containing the resources to update or '-' to read from the standard input.
+   The accepted format is as follow (unchanged values can be omitted):
    {
-     "memory": {
-       "limit": 0,
-       "reservation": 0,
-       "swap": 0,
-       "kernel": 0,
-       "kernelTCP": 0
-     },
-     "cpu": {
-       "shares": 0,
-       "quota": 0,
-       "period": 0,
-       "cpus": "",
-       "mems": ""
-     },
-     "blockIO": {
-       "blkioWeight": 0
-     },
+       "memory": {
+           "limit": 0,
+           "reservation": 0,
+           "swap": 0,
+           "kernel": 0,
+           "kernelTCP": 0
+       },
+       "cpu": {
+           "shares": 0,
+           "quota": 0,
+           "period": 0,
+           "cpus": "",
+           "mems": ""
+       },
+       "blockIO": {
+           "blkioWeight": 0
+       },
```

Comment:
I'm not sure why you moved this to options. I preferred it to be in description TBH.